gnuradio-runtime/include/gnuradio/block.h

```diff
@@ -24,9 +24,9 @@
 #define INCLUDED_GR_RUNTIME_BLOCK_H
 
 #include <gnuradio/api.h>
+#include <gnuradio/logger.h>
 #include <gnuradio/basic_block.h>
 #include <gnuradio/tags.h>
-#include <gnuradio/logger.h>
 
```

Comment:
It is no long necessary to include this here as it would now get pulled in by basic_block.h.